tests/rest/test_new_message_fastapi_endpoint.py

```diff
@@ -13,14 +13,17 @@ def test_new_message_fastapi_post_endpoint_returns_view_model(
     response = httpx_client.post(
         "/conversation/1/message",
         params={
-            "message_content": "Hello, World!",
+            # data={
+            "message_contents": "[Hello, World!]",
             "sender_type": "client",
             "unix_timestamp": 1633096800,
         },
+        json=["body"],
```

Comment:
@maany , any idea why it was erroring unless I added this? Is it because this is a POST request, but we for some reason pass all the arguments as arguments in the query?